valohai_yaml/objs/input.py

```diff
@@ -1,10 +1,27 @@
+from copy import copy
 from enum import Enum
 from typing import List, Optional, Union
 
 from valohai_yaml.objs.base import Item
 from valohai_yaml.types import SerializedDict
 
 KeepDirectoriesValue = Union[bool, str, "KeepDirectories"]
+EdgeMergeModeValue = Union[str, "EdgeMergeMode"]
+
+
+class EdgeMergeMode(Enum):
+    """Input override mode."""
+
+    REPLACE = "replace"
+    APPEND = "append"
+
+    @classmethod
+    def cast(cls, value: EdgeMergeModeValue) -> "EdgeMergeMode":
+        if isinstance(value, EdgeMergeMode):
+            return value
+        if not value:
+            return EdgeMergeMode.APPEND
```

Comment:
The default behavior should be replacing. Is this for the backward compatibility?